docker/test_e2e.sh

```diff
@@ -6,13 +6,12 @@ set -exo pipefail
 
 cat <<EOF > /tmp/ws-expected.log
 OPEN
-CLIENT->SERVER: 1311 (websocket)
-SERVER->CLIENT: 1311 (websocket)
+SERVER->CLIENT: 30 (websocket)
+CLIENT->SERVER: 7 (websocket)
+SERVER->CLIENT: 7 (websocket)
 CLOSE
 EOF
 
-echo "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?" \
-| tr -d \\n \
-| websocat --text -1 ws://${ENDPOINT}
+echo "asdasd" | websocat --text ws://${ENDPOINT}
```

Comment:
Why did you change it?

Reply:
Because:
1. Behavior of the echo server I used (jmalloc/echo-server:0.3.3) is not compatible with -1 options from websocat as this server sends an additional message in the beginning
2. I just made the string smaller to keep it simple and short. There is no need for a full "lorem ipsum ..." thing.
3. I removed `tr -d \\n \` as we don't really need it